modules/representation_management/app/controllers/representation_management/v0/power_of_attorney_controller.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require 'lighthouse/benefits_claims/service'
+
+module RepresentationManagement
+  module V0
+    class PowerOfAttorneyController < ApplicationController
+      service_tag 'representation-management'
+
+      def index
+        begin
+          body = lighthouse_service.get_power_of_attorney
+        rescue BenefitsClaims::ServiceException => e
+          Rails.logger.error("Error fetching power of attorney: #{e.message}")
+          return render json: { error: 'Unable to fetch power of attorney information' }, status: :service_unavailable
+        end
+
+        type = body.dig(:data, :type)
```

Comment:
I'm pretty sure the response body returns string keys, so you'd need to `dig('data', 'type')` or convert the keys. At least that was my experience with the vcr cassette.

Reply:
Updated.